rcl_interfaces/msg/ParameterDescriptor.msg

```diff
@@ -1,8 +1,19 @@
 # This is the message to communicate a parameter's descriptor.
 
 string name
-uint8 type # Enum defined in ParameterType.msg
-# If 'true' then the value cannot change after it has been initialized
+
+# Enum values are defined in the `ParameterType.msg` message.
+uint8 type
+
+# Parameter constraints
+
+# If 'true' then the value cannot change after it has been initialized.
 bool read_only false
 
-# TODO: add additional meta data
+# docs...
```

Comment:
Maybe these don't need docs here? Since they have docs in their own message files.

Reply:
A brief description should be added since it can be represented as the doc string of that member in the generated code (which the comment in the submessage won't provide). It can also clarify why this is a bounded sequence.